pages/who-we-are-hiring.md

```diff
@@ -20,10 +20,10 @@ Everywhere in the United States! If you have no experience working on remote tea
 We're actively hiring for the following positions:
 
 ###Software development (front end)
-Using modern, open source web frameworks and tools, our front and back end developers work in small, cross-functional teams, close to user research and product prioritization, to produce functional, scalable web services.
```

Comment:
@emileighoutlaw: No worries.

@awfrancisco: Thanks for the context. Personally it made a difference to me to see that developers are close to user research and product prioritization, so I may take a shot at reincorporating that in another sentence (or bounce this to one of our new content folks).